api/nsp/v1/utils.go

```diff
@@ -135,7 +139,16 @@ func (f *Flow) DeepEquals(f2 *Flow) bool {
 
 // compareSlice -
 // Returns false if the two slices are not equal by content
+// Intentional? This is more a set-compare. a=[x,x,y], b=[y,y,x] would return true.
```

Comment:
Wasn't intentional :)
However considering the usage it doesn't make a difference.

Reply:
I modify the comment but leave it since there may be more nitpickers than me.